src/mcts/search.h

```diff
@@ -162,6 +165,8 @@ class Search {
   uint16_t max_depth_ GUARDED_BY(nodes_mutex_) = 0;
   // Cummulative depth of all paths taken in PickNodetoExtend.
   uint64_t cum_depth_ GUARDED_BY(nodes_mutex_) = 0;
+  std::atomic<int> tb_hits_;
```

Comment:
Better to do `= 0` here, rather than in constructor.

Reply:
Ah I think jjosh was having problems getting it working and tried various kinds of initialization, but it turned out not to be initialization at all. will fix